.github/workflows/codeowners-validation.yml

```diff
@@ -0,0 +1,58 @@
+name: CODEOWNERS Validation
+run-name: ${{ github.actor }} is checking each Dockerfile for a CODEOWNER
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  CheckDockerfileCodeowners:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+      - name: Check each Dockerfile for a CODEOWNER
```

Comment:
Is it feasible to flag non-existent paths?  E.g. I am envisioning the CODEOWNERS file could become bloated over time as Dockerfiles are removed.  It would be nice to "flag" any obsolete CODEOWNER paths.